conans/test/unittests/model/test_conf.py

```diff
@@ -85,3 +89,28 @@ def test_parse_spaces():
     c = ConfDefinition()
     c.loads(text)
     assert c["core:verbosity"] == "minimal"
+
+
+def test_conf_actions(conf_definition):
+    c, _ = conf_definition
+    text = textwrap.dedent("""\
+    tools.microsoft.msbuild:verbosity=+another
+    user.company.toolchain:flags+=moreflags
+    # tools.cmake:njobs=5
+    # tools.flags:cc=-first 1 -second 2
+    # tools.flags:cc+=-third 3
+    # tools.flags:cxx=-first 1
+    # tools.cmake:njobs=!
+    # tools.flags:cxx=+-second 2
+    # tools.flags:cxx=+-third 3
+    """)
+    c2 = ConfDefinition()
```

Comment:
We need more tests of access, checking what is the final value (via apply() or get()) of a given conf.